docs/adminguide/updating/index.rst

```diff
@@ -0,0 +1,49 @@
+.. Parsec Cloud (https://parsec.cloud) Copyright (c) BUSL-1.1 2016-present Scille SAS
+
+.. _doc_adminguide_updating:
+
+.. cspell:words linenos literalinclude
+
+======================
+Parsec Server updating
+======================
+
+This guide covers the updating procedure for the `Parsec`_ server (also called `parsec-backend`).
+
+.. note::
+
+  We consider that you are using the same infrastructure configuration as in the :ref:`Parsec Server installation guide <doc_adminguide_hosting>`.
+  Adapt this guide to your own situation.
+
+.. _Parsec: https://parsec.cloud
+
+Updating to a newer version
+***************************
+
+.. warning::
+
+  Before updating to a newer version, we recommend that you backup the database,
+  In case you need to rollback the update.
+
+In this guide, we will consider that you are using the parsec-backend at the version ``2023-10-11-v2.16.0-rc.5.dev-b33d909`` and you want to update-it to ``v2.16.0``.
+
+1. Update the docker image's tag of the ``parsec-backend`` in docker-compose's file:
+
+  .. literalinclude:: ../hosting/parsec-server.docker.yaml
+    :language: yaml
+    :linenos:
+    :emphasize-lines: 43
+
+  You need to change the used tag (at line **43**) to ``v2.16.0``.
```

Comment:
This may be prone to errors if someone changes the `../hosting/parsec-server.docker.yaml`. Maybe it is better to just duplicate the line here instead of reference it: there is still the risk of someone updating the docker compose file, but at least the line here won't point to wrong line (it will be just outdated)

Reply:
BTW I've opened a PR to add line numbers in the installation guide too (useful if the installation guide is opened in another tab for example): https://github.com/Scille/parsec-cloud/pull/5760

Reply:
done